news/7230.doc

```diff
@@ -0,0 +1 @@
+This change will change wheel.py to respect docstring conventions.
```

Comment:
I would say "Change \`\`pip._internal.wheel\`\` to respect docstring conventions." - it might even be better as a .trivial news file since it isn't user-facing.

Reply:
Thank you for the suggestions!